language-design.md

```diff
@@ -17,6 +17,18 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
+1. "I should not design a small language, and I should not design a large one. I need to design a language that can grow. I need to plan ways in which it might grow—but I need, too, to leave some choices so that other persons can make those choices at a later time" [Steele, 1998].
+
+   I found this perspective interesting, as it related to some of the in-class discussions we have had about addressing the needs of users and languages. We need to leave room for the users, even though an implementer may be inclined to prioritize the functionality they desire, they should prioritize their language being broadly usable over something strictly holding true to their original image.
```

Comment:
I thought this was interesting too! It will definitely stay on my mind as we start designing DSLs of our own.